sql_request_abstract/models/sql_request_mixin.py

```diff
@@ -144,8 +144,10 @@ def _execute_sql_request(
         if mode in ('view', 'materialized_view'):
             rollback = False
 
-        params = params and params or {}
-        query = self.env.cr.mogrify(self.query, params).decode('utf-8')
+        params = params or {}
+        # pylint: disable=E8103
```

Comment:
Use the name of the lint instead of the number.

Reply:
and what is the name?

Reply:
sql-injection
https://github.com/OCA/pylint-odoo/blob/master/README.rst